contents/docs/custom-mutators.mdx

```diff
@@ -271,7 +271,7 @@ You can get the server result of a mutation with the `server` property of a muta
 const { server } = await zero.mutate.issue.update({
```

Comment:
thanks. no `await` here when getting the server promise iirc though.

![CleanShot 2025-05-01 at 18 46 44@2x](https://github.com/user-attachments/assets/20679ea5-aa31-4318-a9da-39f602ea47cb)